internal/cli/opsmanager/automation/watch.go

```diff
@@ -68,7 +68,11 @@ func WatchBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "watch",
 		Short: "Watch for automation changes to be completed.",
-		Args:  require.NoArgs,
+		Long: `This command checks the automation change's status periodically until they are completed. 
+Once the expected status is reached, the command prints "Changes deployed successfully."
+If you run the command in the terminal, it blocks the terminal session until the changes are completed.
+You can interrupt the command's polling at any time with CTRL-C.`,
+		Args: require.NoArgs,
```

Comment:
```suggestion
		Long: `This command checks the automation configuration's status periodically until they are completed. 
Once the expected status is reached, the command prints "Changes deployed successfully."
If you run the command in the terminal, it blocks the terminal session until the changes are completed.
You can interrupt the command's polling at any time with CTRL-C.`,
		Args: require.NoArgs,
```

Reply:
Feel free to resolve this comment if you update the message as @Gustavo is suggesting 😆